### DISCUSSION TITLE
Feature request - Remote path when adding a Connection

### DISCUSSION THREAD

**Discussion by vladstudio** [mikekazakov/nimble-commander#124] [Ideas]
Would be awesome to have Remote path field here:

<img width="467" alt="image" src="https://github.com/mikekazakov/nimble-commander/assets/914320/fd00468e-3288-47d4-a696-4dd3211c5e5a">

I use multiple folders on my remote servers frequently, and I'd love to be able to have multiple connections, same server but different paths.

Here's how Midnight Commander does it:

<img width="1136" alt="image" src="https://github.com/mikekazakov/nimble-commander/assets/914320/1ed497b1-4856-4b90-918e-dccf97fece84">

**Comment by mikekazakov** [comment]
Hm, that what Favourites are presumably for?

**Comment by vladstudio** [comment]
Oh I see! Yes, I was able to connect to server, go to a folder, and add it to Favourites. Did not think I could do that!
